RELEASE.md

```diff
@@ -20,6 +20,21 @@
 
 ## Documentation changes
 * Added documentation to clarify execution order of hooks.
+* Added a notebook example for spaceflights to illustrate how to incrementally add Kedro features.
+* Moved documentation for the `standalone-datacatalog` starter into its [README file](https://github.com/kedro-org/kedro-starters/tree/main/standalone-datacatalog).
+* Added new documentation about deploying a Kedro project with Amazon EMR.
+* Added new documentation about how to publish a Kedro-Viz project to make it shareable.
+* New TSC members added to the page and the organisation of each member is also now listed.
+* Plus some minor bug fixes and changes across the documentation.
+
+## Community contributions
+Many thanks to the following Kedroids for contributing PRs to this release:
+
+* [Laíza Milena Scheid Parizotto](https://github.com/laizaparizotto)
```

Comment:
@stichbury There's another Community contributions section below (after the deprecation notices) with some more people, maybe we should combine them?